<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Shots Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="shotsPathBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="statusBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addAssetsButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Add assets from TACTIC for the specified Sequence</string>
        </property>
        <property name="text">
         <string>Sequence Assets</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resolutionBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="contactShadowButton">
        <property name="toolTip">
         <string>Create ContactShadow layer from shadow layer</string>
        </property>
        <property name="text">
         <string>Contact Shadow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="envOccButton">
        <property name="toolTip">
         <string>Create Env_Occ layer from Env layer</string>
        </property>
        <property name="text">
         <string>Env Occ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="occButton">
        <property name="toolTip">
         <string>Create Occ layer from Shadow layer</string>
        </property>
        <property name="text">
         <string>Occ</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>536</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
